src/pages/api/dev_cardholder.ts

```diff
@@ -0,0 +1,32 @@
+import { NextApiRequest, NextApiResponse } from "next";
```

Comment:
This will eventually include more logic to process cardholder commands, right?

Reply:
Yeah, it was just a mock page to ensure api + prisma was working. I've deleted it since it works